packages/main/src/RadioButton.js

```diff
@@ -174,8 +175,7 @@ const metadata = {
 		},
 
 		/**
-		 * Defines the text alternative of the component.
-		 * If not provided a default text alternative will be set, if present.
+		 * Defines the accessible name of the component.
```

Comment:
I am not sure that this is self explanatory enough

Reply:
I have taken this sentence from the other components.  The second sentence "If not provided a default text ..." is not completely true, so I removed it.
I think we can add something like in Openui5 - https://openui5.hana.ondemand.com/api/sap.m.RadioButton#associations **"see WAI-ARIA attribute aria-label"**. But we should do this for all components or not at all. What do you think?

Reply:
It is explanatory enough. What is more, additional information could be found in the accessibility documentation.